src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.quarto.component.spec.ts

```diff
@@ -1049,6 +1119,143 @@ describe('OnlineGameWrapperComponent of Quarto:', () => {
         it('when resigning, lastMoveTime must be upToDate then remainingMs');
         it('when winning move is done, remainingMs at last turn of opponent must be');
     });
+    describe('AddTime functionnalities', () => {
+        it('should allow to add local time to opponent', fakeAsync(async() => {
+            // Given an onlineGameComponent
+            await prepareStartedGameFor(USER_CREATOR);
+            spyOn(partDAO, 'update').and.callThrough();
+            tick(1);
+
+            // when local countDownComponent emit addTime
+            await wrapper.addLocalTime();
+
+            // then some kind of addLocalTimeTo(player) should be sent
+            expect(partDAO.update).toHaveBeenCalledOnceWith('joinerId', {
+                request: Request.localTimeAdded(Player.ONE),
```

Comment:
ce GWT laisse à désirer non ? (Les commentaires correspondent pas trop au code, ou de façon trop cachée en tout cas)

Reply:
// Then partDao should have send a Request.localTimeAdded ?

Reply:
should be updated with a Request.localTimeAdded ? mais oui sinon

Reply:
then should ? Sinon comme title c'est vachement trop spécifique non ?

Reply:
oui oui `// Then partDAO should be updated with a Request.localTimeAdded`, pas should du titre